src/Chainweb/Chainweb.hs

```diff
@@ -442,7 +448,6 @@ data Chainweb logger cas = Chainweb
     , _chainwebCutResources :: !(CutResources logger cas)
     , _chainwebMiner :: !(Maybe (MinerResources logger cas))
     , _chainwebCoordinator :: !(Maybe (MiningCoordination logger cas))
-    , _chainwebHeaderStream :: !HeaderStream
```

Comment:
This is a redundant field - the setting itself is already contained in `ChainwebConfiguration`.